From a3e8b5d47dd607d47237cfb38c998ea1dd03f6ce Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Fri, 14 Aug 2015 14:55:58 +0800
Subject: [PATCH 0210/5242] MLK-11363-1 ARM: dts: imx7d-12x12-lpddr3-arm2:
 enable epxp in device node

commit  ebf1561497d15aafc4afbe2f1d1fbad7a24309d3 from
https://source.codeaurora.org/external/imx/linux-imx.git

Enable epxp in device node by default on imx7d-12x12-lpddr3-arm2
board.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
index ba91476..ddb1cd0 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
@@ -80,6 +80,10 @@
 	arm-supply = <&sw1a_reg>;
 };
 
+&epxp {
+	status = "okay";
+};
+
 &ecspi1 {
 	fsl,spi-num-chipselects = <1>;
 	cs-gpios = <&gpio4 19 0>;
-- 
1.7.9.5

